### DISCUSSION TITLE
Display ongoing running jobs in the mend renovate bot dashboard

### DISCUSSION THREAD

**Discussion by gberche-orange** [renovatebot/renovate#22369] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

The mend renovate bot user experience UX is great (ability to trigger/restart jobs from the GH issue). 

The dashboard displays the completed jobs, but not the currently ongoing jobs
![image](https://github.com/renovatebot/renovate/assets/4748380/6eefb4a5-af83-4a2d-a6dc-c6f1bd3ec70d)

The current lack of visibility into currently running jobs causes the following issues:
* frenetic dashboard refreshes to see completed jobs appear
* difficult to know whether a job was already triggered or not, especially for long jobs, and which commit where cloned from the repo
* manually blindly triggering extra duplicate jobs (which waste computing resources)

It would make sense to flush a 1st chunk of job log in the first few seconds after the `DEBUG: latest commit` to know the job was properly

**Comment by rarkins** [comment]
Good news! We are in the process of switching to a new backend - hopefully in the next week - and it will be accompanied also with a new web UI. The UI will have very simple feature parity to begin with (view orgs, repos, logs) but one exception is that it will be able to show jobs in queue and in progress. We'll redirect the current UI (app.renovatebot.com) to the new one once we're confident.

**Reply by gberche-orange** [reply]
thanks a lot @rarkins !

It is very useful in the new UI to see the currently job running !

![image](https://github.com/renovatebot/renovate/assets/4748380/94f631f7-4f2e-4251-99f3-12370d9b36f9)
